storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/V4FinalizedKvStoreDao.java

```diff
@@ -128,14 +182,18 @@ public FinalizedUpdater finalizedUpdater() {
     return new V4FinalizedKvStoreDao.V4FinalizedUpdater(db, schema, stateStorageFrequency);
   }
 
-  private static class V4FinalizedUpdater implements FinalizedUpdater {
+  static class V4FinalizedUpdater implements FinalizedUpdater {
     private final KvStoreTransaction transaction;
     private final KvStoreAccessor db;
     private final SchemaFinalized schema;
     private final UInt64 stateStorageFrequency;
     private Optional<UInt64> lastStateStoredSlot = Optional.empty();
     private boolean loadedLastStoreState = false;
 
+    KvStoreTransaction getTransaction() {
+      return transaction;
+    }
```

Comment:
I suspect we shouldn't be exposing this.  I'd just provide a putRaw method here.

Reply:
Ignore this, it turns out we don't need to use the updaters at all.

Reply:
removed, you're 100% correct, it's not needed now :)